### DISCUSSION TITLE
Detail Requirements Installing Templates

### DISCUSSION THREAD

**Discussion by clayjam** [microsoft/Templates-for-Power-Platform#50] [General]
Can you please detail any requirements needed to install these templates? 

For example, for the Kudos app do we need Dataverse for this work or can we use SharePoint lists.

I also read somewhere these apps require an installation of a core app.

**Comment by mhuguet** [comment]
Each template requires Dataverse and the end users will need premium Power Apps licensing for the app templates. For Copilot templates, the user will either need Copilot Studio messages or Copilot for M365.
